mlir/include/mlir/Dialect/Rock/Tuning/GridwiseGemmParams.h

```diff
@@ -309,16 +311,14 @@ class PopulateParamsAccel : public BasePopulateParams<InitParamsAccel> {
   static std::unique_ptr<PopulateParamsAccel> select(GemmFeatures features);
 
   LogicalResult obtainTuningParameters(RockGemmWrapperInterface op,
-                                       uint32_t blockSizeOverride,
                                        const std::string &perfConfig,
                                        InitParamsAccel &validParams,
-                                       uint32_t &blockSize, uint32_t &gridSize,
-                                       int64_t &gemmKBlocks);
+                                       uint32_t &blockSize);
 
-  virtual LogicalResult obtainTuningParameters(
-      const PopulateParamsInfo &info, uint32_t blockSizeOverride,
-      const std::string &perfConfig, InitParamsAccel &validParams,
-      uint32_t &blockSize, uint32_t &gridSize, int64_t &gemmKBlocks);
+  virtual LogicalResult obtainTuningParameters(const PopulateParamsInfo &info,
+                                               const std::string &perfConfig,
```

Comment:
Can we make this a `StringRef` while we're here?

Reply:
Yes. Done